server/protocols/jmap/src/main/java/org/apache/james/jmap/model/JmapFlag.java

```diff
@@ -0,0 +1,49 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.model;
+
+import javax.mail.Flags;
+
+public enum JmapFlag {
+    $Draft(Flags.Flag.DRAFT),
+    $Seen(Flags.Flag.SEEN),
+    $Flagged(Flags.Flag.FLAGGED),
+    $Answered(Flags.Flag.ANSWERED),
+    $Forwarded(Flags.Flag.USER)
+    ;
+    private Flags.Flag imapFlag;
+
+    JmapFlag(Flags.Flag imapFlag) {
+        this.imapFlag = imapFlag;
+    }
+
+    public Flags.Flag getImapFlag() {
```

Comment:
Flags.Flag is not immutable (or at least you can't rely on it) so you should not expose it without a copy. An Enum entry required to be immutable.